src/UI/class-network-admin-sites-list.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Parsely Network Admin Site List class
+ *
+ * @package Parsely
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+use Parsely\Parsely;
+
+/**
+ * Render the additions to the WordPress Multisite Network Admin Sites List page
+ *
+ * @since 3.2.0
+ */
+final class Network_Admin_Sites_List {
+	/**
+	 * Constructor.
+	 *
+	 * @param Parsely $parsely Instance of Parsely class.
+	 */
+	public function __construct( Parsely $parsely ) {
+		$this->parsely = $parsely;
+	}
+
+	/**
+	 * Attach network admin page functionality to the appropriate action and filter hooks.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return void
+	 */
+	public function run() {
+		add_filter( 'manage_sites_action_links', array( __CLASS__, 'add_action_link' ), 10, 2 );
+		add_filter( 'wpmu_blogs_columns', array( __CLASS__, 'add_api_key_column' ) );
+		add_action( 'manage_sites_custom_column', array( $this, 'populate_api_key_column' ), 10, 2 );
+	}
+
+	/**
+	 * Use the manage_sites_action_links filter to append a link to the settings page in the "row actions."
+	 *
+	 * @param array $actions The list of actions meant to be displayed for the current site's context in the row actions.
+	 * @param int   $_blog_id The blog ID for the current context.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return array The list of actions including ours.
+	 */
+	public static function add_action_link( array $actions, int $_blog_id ): array {
+		if ( ! current_user_can( Parsely::CAPABILITY ) ) {
+			return $actions;
+		}
+
+		if ( ! is_numeric( $_blog_id ) ) {
+			return $actions;
+		}
+
+		$settings_url                = Parsely::get_settings_url( (int) $_blog_id );
```

Comment:
Blog id should always be an integer if we're using strict types, so I think we don't need the casting here.

Reply:
Ah, cool.  I didn't realize that adding the `int` type declaration still coerces a numeric string to an `int` with strict types declared. :TIL:

I had encountered a TypeError prior to adding the type declarations because the function was passed a string for the blog id.

That means I can take the `is_numeric` check out above.  Neat.